lib/routes/index.js

```diff
@@ -87,6 +94,74 @@ module.exports = function routing(app, options, callback) {
       });
     });
 
+    /**
+     * @swagger
+     * /:
+     *   get:
+     *     summary: Gets high-level information about all packages
+     *     produces:
+     *       - "text/html; charset=utf-8"
+     *     responses:
+     *       200:
```

Comment:
this needs to support 302 for auth middleware redirect

Reply:
ah. good catch. Will add it to all routes, probably add in 401/403/404 as well for everything to be safe